requirements.txt

```diff
@@ -0,0 +1,4 @@
+asgiref==3.4.1
+Django==3.2.5
+pytz==2021.1
+sqlparse==0.4.1
```

Comment:
I mentioned this in the PR comment, but I think we probably want to manage our Python dependencies differently than a requirements.txt file.